### ISSUE TITLE
Object coordinates missing

### ISSUE THREAD

**Issue by thomashenn** [kimhc6028/relational-networks#4]
From the article in the "Dealing with pixels" case:
>  So, after convolving the image, each of the d^2 k-dimensional cells in the d × d feature maps was tagged with an arbitrary coordinate indicating its relative spatial position, and was treated as an object for the RN.

Also, the author (/u/asantoro) confirmed on reddit that objects were of the form:
[x, y, v_1, v_2, ..., v_k] where k is the number of kernels and the range of the coordinates x,y doesn't matter.
[(Reddit link)](https://www.reddit.com/r/MachineLearning/comments/6fk2i6/r_170601427_from_deepmind_a_simple_neural_network/dij3zph/)

So I think in the model, object coordinates should be added to oi and oj.
https://github.com/kimhc6028/relational-networks/blob/master/model.py#L53
```python
for i in range(25):
    oi = x[:,:,i/5,i%5]
    for j in range(25):
        oj = x[:,:,j/5,j%5]
        x_ = torch.cat((oi,oj,qst), 1)
        x_ = self.g_fc1(x_)
```        
I believe this should improve performance on questions where the spatial relationship between objects is important (closest, furthest, ...).

**Comment by kimhc6028** [comment]
Oops, thank you for pointing out my mistake. I am working with it with first priority. Please wait a bit...

**Comment by kimhc6028** [comment]
It seems object coordinates improves accuracy. The code is now fixed. Thank you!

**Comment by thomashenn** [comment]
Glad to hear that ! Thank you :+1:
